webapp/src/shared/components/launcher/background/background.component.js

```diff
@@ -0,0 +1,14 @@
+import { Container, Sprite, Texture } from 'pixi.js';
+
+class Background extends Container {
+  constructor({ image }) {
+    super();
+    this.texture = Texture.from(image);
+    this.image = new Sprite(this.texture);
+    this.image.width = 1;
```

Comment:
why 1?

Reply:
Well, this value does not impact to anything, but lack of it finishes with not displayed image at all. 😕